blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -370,9 +368,7 @@ class ResultsList extends Component {
                 )}
                 { (promoElements.showDate || promoElements.showByline) && (
                 <div className="results-list--author-date">
-                  { promoElements.showByline && <Byline story={element} stylesFor="list" /> }
-                  {/* The Separator will only be shown if there is at least one author name */}
-                  { promoElements.showByline && showSeparator && promoElements.showDate && <p className="dot-separator">&#9679;</p> }
+                  { promoElements.showByline && <Byline story={element} stylesFor="list" separator={promoElements.showDate} /> }
```

Comment:
We want to ensure this `<Byline` component is the shared styles component being used not the arc block version